charts/keycloakx/values.yaml

```diff
@@ -275,15 +275,15 @@ ingress:
       host: '{{ .Release.Name }}.keycloak.example.com'
       # Paths for the host
       paths:
-        - path: /
+        - path: '{{ trimSuffix "/" .Values.http.relativePath}}/'
```

Comment:
I am not sure if this is correct. The ingress should forward all requests to the keycloak service, regardless of the fact that there might be a relativePath configured or not, right?

Reply:
Fix